vault/migration-bitwarden.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+
+if [[ -z $VAULT_TOKEN ]]; then
+  echo "Please set the varaible $VAULT_TOKEN"
+  exit 1
+fi
+if [[ -z $VAULT_ADDR ]]; then
+  echo "Please set the varaible $VAULT_ADDR"
+  exit 1
+fi
+
+if [[ -z "${@}" ]]; then
```

Comment:
```suggestion
if [[ -z "${*}" ]]; then
```
https://www.shellcheck.net/wiki/SC2199

OR

you could also check if 3 arguments were passed or not like this : 

```suggestion
if [[ $# -ne 3 ]]; then
```